### ISSUE TITLE
Document about masking does not have previous masks information

### ISSUE THREAD

**Issue by hoangcuong2011** [tensorflow/tensorflow#34966]
Greetings,

While the document about [masking](https://www.tensorflow.org/guide/keras/masking_and_padding) is super good, I found it misses an important point: how the mask associated with the previous mask in compute_mask(input, previous_mask)

Specifically, let us assume we have two inputs A and B. I wrote a custom Add layers:

```
class CustomAddingWithMasking(tf.keras.layers.Layer):
    def __init__(self, masking_boolean, **kwargs):
        super(CustomAddingWithMasking, self).__init__(**kwargs)

    def call(self, inputs):
        return inputs[0] + inputs[1]
    
    def compute_mask(self, inputs, mask=None):
        return mask
```

Here, I want to compute the sum of two tensors. Let us also assume that A and B have their own masks, which could be different to each other. Because we technically have two "previous masks" (from A and B separately), I don't know how the mask parameter in compute_mask was received. Is it the OR (or AND?) operation between mask of A and mask of B?

Those things are not clear as well as not documented well.

**Comment by Saduf2019** [comment]
@hoangcuong2011 
Is this still an issue.

**Comment by ymodak** [comment]
The Masking and padding [guide](https://www.tensorflow.org/guide/keras/masking_and_padding) has been updated over time.  Please comment if further information/clarification is required. Thanks!
